src/jvmMain/kotlin/io/beatmaps/api/users.kt

```diff
@@ -639,7 +667,16 @@ fun Route.userRoute() {
             (User.id eq it.id) and User.active
         }
 
-        val userDetail = UserDetail.from(user, stats = statsForUser(user)).let {
+        val following = call.sessions.get<Session>()?.let { sess ->
+            if (it.id == sess.userId) null
+            else transaction {
+                Follows.select {
```

Comment:
Would make sense to me to combine this and the above query (at least into one transaction, but one query also seems trivial)